test/long_lines_code.md

```diff
@@ -29,6 +29,7 @@ This is a short line.
 | Content Cell   | Content Cell   | Content Cell   | Content Cell   | Content Cell   | Content Cell   |
 | ============== | ============== | ============== | ============== | ============== | ============== |
 | Footer Cell    | Footer Cell    | Footer Cell    | Footer Cell    | Footer Cell    | Footer Cell    |
+
 {: rules="groups"}
```

Comment:
If you don't know what this syntax is trying to do, let's maybe delete it?

Reply:
Sure, :smile:,

Reply:
Sure, :smile:,